extensions/core/interfaces/calendar/interface.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<div v-bind:class="{ inactive: readonly }">
+		<flat-pickr class="form-control"
+                :config="configs"
+                v-model="value"
```

Comment:
This throws an error. `value` is being passed in as a prop from the parent, and is being updated by `$emit`ing the `input` event with a value.